napari_animation/animation.py

```diff
@@ -116,6 +116,40 @@ def _set_viewer_state(self, state):
 
         self.viewer.camera.update(camera_state)
         self.viewer.dims.update(state['dims'])
+        self._set_layer_state(state['layers'])
+
+    @property
+    def layers(self):
+        return self.viewer.layers
+
+    def _layer_names(self):
+        return [layer.name for layer in self.layers]
+
+    @property
+    def layer_attributes(self):
```

Comment:
Can we make this private, I think in the long run as we add things like Image or Points specific attributes we'll probably need to change how we do this, so having private seems better